### ISSUE TITLE
Add specificExceptions checking to isValidChild()

### ISSUE THREAD

**Issue by KatieProchilo** [MikeHopcroft/PrixFixe#63]
In rule_checker's isValidChild(), add specificExceptions checking.

**Comment by SpaceKatt** [comment]
This issue depends on #64

**Comment by KatieProchilo** [comment]
Is this a duplicate of #58?
